packages/api/src/beacon/routes/validator.ts

```diff
@@ -443,7 +443,12 @@ export function getReturnTypes(): ReturnTypes<Api> {
     getSyncCommitteeDuties: ContainerDataExecutionOptimistic(ArrayOf(SyncDuty)),
     produceBlock: ContainerData(ssz.phase0.BeaconBlock),
     produceBlockV2: WithVersion((fork: ForkName) => ssz[fork].BeaconBlock),
-    produceBlindedBlock: WithVersion((_fork: ForkName) => ssz.bellatrix.BlindedBeaconBlock),
+    produceBlindedBlock: WithVersion((fork: ForkName) => {
+      if (fork === ForkName.phase0 || fork === ForkName.altair) {
```

Comment:
Use ForkSeq and < ForkSeq.bellatrix comparator

Reply:
Same for `packages/config/src/forkConfig/index.ts`

Reply:
in `packages/config/src/forkConfig/index.ts` typescript can't infer with `seq` comparision that fork is not pre bellatrix causing type issues for accessing `ssz.allForksBlinded[forkName]`

Reply:
also same here as well

Reply:
ok figure it out thanks!

Reply:
duplicated the bellatrix blinded types for phase0 and altair for type consistency, the check here is no more required, while the check in  `packages/config/src/forkConfig/index.ts` is now using `seq` comparision. let me know if this approach is good

Reply:
Oh right, yeah then it's okay typesafety always wins